Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -686,10 +706,11 @@ def fetch_incidents():
                 if incident_date_timestamp > last_fetch_timestamp:
                     last_fetch = incident_date
                     last_fetch_timestamp = incident_date_timestamp
+                    last_detection_id = json.loads(incident['rawJSON']).get('detection_id')
```

Comment:
is it numerical?

Reply:
no - it's a string with both letters and numbers